samtranslator/internal/model/appsync.py

```diff
@@ -25,19 +25,27 @@ class DynamoDBConfigType(TypedDict, total=False):
     DeltaSyncConfig: DeltaSyncConfigType
 
 
+class LogConfigType(TypedDict, total=False):
+    CloudWatchLogsRoleArn: Intrinsicable[str]
```

Comment:
We don't do intrinsicable anymore

Reply:
If it's to static-type the output fields and intrinsics are used it's fine/needed.

Reply:
I thought we discussed that for the CFN types they were fine to use?